working/1855 - super parameters/proposal.md

```diff
@@ -46,9 +46,9 @@ _That is, exactly the same grammar as initializing formals, but with `super` ins
 
 ## Semantics
 
-It’s a **compile-time error** if a super-parameter in any declaration other than a non-redirecting generative constructor.
+It’s a **compile-time error** if a super-parameter in any declaration other than a non-redirecting generative constructor of a `class` declaration.
```

Comment:
```suggestion
It’s a **compile-time error** if a super-parameter declaration occurs in any declaration other than a non-redirecting generative constructor of a class declaration.
```

(I think there's ample tradition for using 'class' with no special font, so we might as well keep the visuals simple).